core/src/main/java/org/elasticsearch/indices/InvalidAliasNameException.java

```diff
@@ -29,10 +29,11 @@
 /**
  *
  */
-public class InvalidAliasNameException extends IndexException {
+public class InvalidAliasNameException extends ElasticsearchException {
```

Comment:
can we get rid of this exception? especially now that we have the fancy formatting.